data/json/recipes/recipe_others.json

```diff
@@ -326,6 +326,39 @@
     ]
   },{
   "type" : "recipe",
+  "result": "tailors_kit",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_TOOLS",
+  "skill_used": "tailor",
+  "difficulty": 3,
+  "time": 15000,
+  "reversible": true,
+  "autolearn": false,
+  "book_learn": [[ "textbook_tailor", 3 ] , [ "manual_tailor", 3 ]],
+  "tools": [
+     [
+      ["soldering_iron", -5]
```

Comment:
This should probably be a positive number to make it consume charges. Negative numbers indicates item counts.